src/front-door/azext_front_door/tests/latest/test_route_rule_scenarios.py

```diff
@@ -22,7 +22,7 @@ def test_route_rule_basic(self, resource_group):
                  '--patterns /forward2 --caching Disabled',
                  checks=[
                      self.check('patternsToMatch[0]', '/forward2'),
-                     self.check('contains(keys(routeConfiguration), "cacheConfiguration")', False)
+                     self.check('routeConfiguration.cacheConfiguration', None)
```

Comment:
is there any reason to change it?

Reply:
When I re-ran this test, ```keys(routeConfiguration)``` was returning 'None'. Hence ```contains(keys(routeConfiguration), "cacheConfiguration")``` would be returning False regardless.
```routeConfiguration.cacheConfiguration``` check was correctly testing for cache being disabled